dart/biomechanics/DynamicsFitter.cpp

```diff
@@ -11618,7 +11618,12 @@ bool DynamicsFitter::zeroLinearResidualsOnCOMTrajectory(
     //     unifiedPositions - unifiedGravityOffset);
     Eigen::LeastSquaresConjugateGradient<Eigen::MatrixXs> solver;
     solver.setTolerance(1e-9);
-    solver.setMaxIterations(unifiedLinearMap.rows() * 10);
+    int maxIterations = unifiedLinearMap.rows() * 10;
+    if (maxIterations > 1000)
+    {
+      maxIterations = 1000;
+    }
```

Comment:
Capping at a 1000 iterations shouldn't change much right? This usually converges in a handful of iterations if everything is set up right.